core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -85,18 +82,13 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Loggi
       "-baseport", conf.nodeBasePort.toString,
       "-timeout", conf.clusterStartTimeout.toString,
       "-disown",
-      "-sw_ext_backend"
+      "-sw_ext_backend",
+      "-J", "-rest_api_ping_timeout", "-J", conf.clientCheckRetryTimeout.toString
     )
 
-    if (isRestApiBasedClient(hc)) {
-      cmdToLaunch = cmdToLaunch ++ Seq[String]("-J", "-rest_api_ping_timeout", "-J", conf.clientCheckRetryTimeout.toString)
-    } else {
+    if (!isRestApiBasedClient(hc)) {
       cmdToLaunch = cmdToLaunch ++ Seq[String](
-        "-J", "-client_disconnect_timeout", "-J", conf.clientCheckRetryTimeout.toString,
-        "-J", "-watchdog_stop_without_client",
```

Comment:
no need for watchdog, we have the rest api check we can use also in regular client mode of external backend